roles/edpm_nova/defaults/main.yml

```diff
@@ -31,7 +31,10 @@ edpm_nova_compute_image: "quay.io/podified-antelope-centos9/openstack-nova-compu
 
 # Libvirt TLS
 edpm_nova_live_migration_tls: "{{ edpm_tls_certs_enabled | default(False) }}"
-edpm_nova_live_migration_native_tls: "{{ edpm_tls_certs_enabled | default(False) }}"
+# FIXME(owalsh): Disable NBD TLS until issues are resolved:
+#   https://issues.redhat.com/browse/RHEL-33754
+#   https://issues.redhat.com/browse/RHEL-33781
+edpm_nova_live_migration_native_tls: false
```

Comment:
we spoke about this on slack and for beta this is what we should do until we move to rhel 9.4 with the fixes for the two referenced issues.

for ga we will hopefully be able to revert to enabling edpm_nova_live_migration_native_tls when tls is enabeld.